detections/application/papercutng_suspicious_behavior_debug_log.yml

```diff
@@ -0,0 +1,58 @@
+name: PaperCutNG Suspicious Behavior Debug Log
+id: 395163b8-689b-444b-86c7-9fe9ad624734
+version: 1
+date: '2023-05-15'
+author: Michael Haag, Splunk
+status: production
```

Comment:
can we move this detection to experimental since we don have proper parses written for this sourcetype?